store/conf/attrs/zimbra-attrs.xml

```diff
@@ -1084,7 +1084,7 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
   <desc>whether or not to index attachments</desc>
 </attr>
 
-<attr id="174" name="zimbraImapEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable">
+<attr id="174" name="zimbraImapEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable">
```

Comment:
why do you need this in accountinfo?

Reply:
Fixed.

Reply:
Does the following code work without accountInfo flag? The settings should appear only when zimbraImap(Pop3)Enabled is TRUE.

https://github.com/Zimbra/zm-web-client/pull/615/files#diff-2dfc4b8571a8d8eed10c5e28f83862137467b947f993728d08f75cfc6f364d5d

	ZmPref.registerPref("PREF_POP_ENABLED", {
		displayName:		ZmMsg.popAccess,
		displayContainer:	ZmPref.TYPE_CHECKBOX,
		precondition:		[ZmSetting.POP_ENABLED]
	});

	ZmPref.registerPref("PREF_IMAP_ENABLED", {
		displayName:		ZmMsg.imapAccess,
		displayContainer:	ZmPref.TYPE_CHECKBOX,
		precondition:		[ZmSetting.IMAP_ENABLED]
	});

Reply:
Yes. it will work as now we don't facilitate admin to configure additional attributes which prashant ask to remove.

Reply:
I checked the behavior on my local vm; "Enable IMAP access" setting was shown in Preferences -> Mail page even when zimbraImapEnabled was FALSE. It was unexpected behavior. Then "accountInfo" flag is necessary on zimbraImapEnabled.

Reply:
Note that the change of the flag will not affect ldap schema. Regarding a patch release, the change will be applied just replacing/updating zimbra-attrs.xml.

Reply:
Fixed.

Reply:
In GetInfo response to can check and send only one attribute. 
zimbraPrefImapEnabled  = zimbraImapEnabled && zimbraPrefImapEnabled

Reply:
Fixed.